go/consensus/tendermint/apps/scheduler/scheduler.go

```diff
@@ -393,7 +393,7 @@ func (app *schedulerApplication) isSuitableTransactionScheduler(n *node.Node, rt
 }
 
 func (app *schedulerApplication) isSuitableMergeWorker(n *node.Node, rt *registry.Runtime, ts time.Time) bool {
-	return n.HasRoles(node.RoleMergeWorker)
+	return n.HasRoles(node.RoleComputeWorker)
```

Comment:
This (and storage) needs to be made per-runtime (as mentioned in https://github.com/oasislabs/oasis-core/issues/2107#issuecomment-570572948), similar to txnscheduler/compute.

Reply:
ok I added the runtime snippet to these